piksi_tools/console/baseline_view.py

```diff
@@ -16,7 +16,7 @@
 from enable.api import ComponentEditor
 from enable.savage.trait_defs.ui.svg_button import SVGButton
 from pyface.api import GUI
-from utils import plot_square_axes
+from piksi_tools.console.utils import plot_square_axes
 
```

Comment:
What's wrong with the relative imports?  I find it very hard to develop when absolute imports are used and you have a version installed with setup.py on your machine.  Is this where setup.py develop is useful?